llvm/lib/Target/X86/X86InstrAMX.td

```diff
@@ -369,3 +369,150 @@ let Predicates = [HasAMXTRANSPOSE, In64BitMode] in {
     }
   }
 } // HasAMXTILE, HasAMXTRANSPOSE
+
+multiclass m_tcvtrowd2ps {
+  let Predicates = [HasAMXAVX512, In64BitMode] in {
```

Comment:
Should add HasAVX10_2_512 in line 374, 390 and 454?

Reply:
Done, thanks!